internal/locate/pd_codec.go

```diff
@@ -85,6 +85,15 @@ func GetKeyspaceID(client pd.Client, name string) (uint32, error) {
 	return meta.Id, nil
 }
 
+// GetKeyspaceMeta attempts to retrieve keyspace meta corresponding to the given keyspace name from PD.
+func GetKeyspaceMeta(client pd.Client, name string) (*keyspacepb.KeyspaceMeta, error) {
```

Comment:
If `GetKeyspaceID` is still used in some external package, we might implement it using `GetKeyspaceMeta`.